src/datachain/data_storage/metastore.py

```diff
@@ -851,6 +855,24 @@ def get_project(
             )
         return self.project_class.parse(*rows[0])
 
+    def get_project_by_id(self, project_id: int, conn=None) -> Project:
```

Comment:
a bit separate topic - btw, why are we using IDs still, not UUIDs for all these operations?

Reply:
no, we should be using uuids I think ... it's a pitty we didn't start using them originally ... we'll have a very major and heavy migration later